docs/sc_FunctionalRequirements.csv

```diff
@@ -135,15 +135,15 @@ p)       RTS table activation count
 q)       RTS table activation error count
 r)        Number of active RTSs
 s)       Identifier of the next RTS table to dispatch a command
-t)        Time the next RTS command is due to be dispatched
+t)        Wakeup count the next RTS command is due to be dispatched

```

Comment:
Wording is a little weird here, maybe "Wakeup count delay after which the next RTS command..."?